tests/integration/test_stampipes.py

```diff
@@ -3,8 +3,19 @@
 from pathlib import Path
 
 
-@pytest.mark.workflow(name='test_flag_qc_fail_improper_pair_and_nonnuclear_bam_reads')
-def test_flag_qc_fail_improper_pair_and_nonnuclear_bam_reads_bams_match(test_data_dir, workflow_dir, bams_match):
-    actual_bam_path = workflow_dir / Path('test-output/flagged.bam')
-    expected_bam_path = test_data_dir / Path('dnase/alignment/flagged.bam')
+@pytest.mark.workflow(name="test_flag_qc_fail_improper_pair_and_nonnuclear_bam_reads")
+def test_flag_qc_fail_improper_pair_and_nonnuclear_bam_reads_bams_match(
+    test_data_dir, workflow_dir, bams_match
```

Comment:
Not a fan of this formatting. I think this is still acceptable length to be on one line.

Reply:
ok, this is black. I'll turn it off.